instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```diff
@@ -0,0 +1,262 @@
+---
+rules:
+  - bean: org.apache.camel:context=*,type=context,name=*
+    prefix: camel.context.
+    metricAttribute:
+      context: param(context)
+      camelVersion: beanattr(CamelVersion)
+    mapping:
+      ExchangesCompleted:
+        metric: exchange.completed
+        type: counter
+        unit: "{exchanges}"
```

Comment:
probably this should be singular `{exchange}`? (check out https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#instrument-units)

Reply:
ok but in the the other rules, unit are all plural. 

Here some few examples:


https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fcc0049aa27f84a6c9452766612e8e2448a40216/instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/tomcat.yaml#L106

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fcc0049aa27f84a6c9452766612e8e2448a40216/instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/wildfly.yaml#L65

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fcc0049aa27f84a6c9452766612e8e2448a40216/instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/kafka-broker.yaml#L11

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fcc0049aa27f84a6c9452766612e8e2448a40216/instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/kafka-broker.yaml#L21

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fcc0049aa27f84a6c9452766612e8e2448a40216/instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/kafka-broker.yaml#L128

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fcc0049aa27f84a6c9452766612e8e2448a40216/instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/kafka-broker.yaml#L156

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fcc0049aa27f84a6c9452766612e8e2448a40216/instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/kafka-broker.yaml#L171

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/fcc0049aa27f84a6c9452766612e8e2448a40216/instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/kafka-broker.yaml#L179

do you still want me to change it or should I leave it as it is for consistency?

Reply:
> Also exchange.completed and all the other metrics where I used "exchanges" as unit, are countable quantities:
> 
> > Metric names SHOULD NOT be pluralized, unless the value being recorded represents discrete instances of a [countable quantity](https://wikipedia.org/wiki/Count_noun).

this guidance is for metric name pluralization, not unit pluralization

I'd prefer to follow the semconv guidance for new metrics, @PeterF778 wdyt?

Reply:
As @luigidemasi pointed out, we use units in plural for most if not all JMX metrics supported OOTB by JMX Metric Insight. And we have the same story for JMX Metric Gatherer (in java-contrib). So, the question is whether we want to prioritize semantic conventions compliance over consistency. Perhaps the best approach would be to still let the new metrics use pluralized units, and revert to singular form of units for ALL metrics in the next major release?

Reply:
thanks, makes sense👍